packages/addons/script/steamlink-rpi/source/default.py

```diff
@@ -0,0 +1,85 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2019-present Team LibreELEC (https://libreelec.tv)
+
+import os
+import tarfile
+import subprocess
+import xbmcaddon
+import xbmcgui
+from hashlib import sha256
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from sys import exit
+from urllib.request import urlretrieve
+
+
+STEAMLINK_VERSION = "1.1.45.105"
```

Comment:
it is quite ugly to have the version/hash at an unusual place, could we proxy in some not hacky way the version/hash to the package.mk ?
or are there a "latest.zip" somewhere ?

Reply:
http://media.steampowered.com/steamlink/rpi/public_build.txt holds the URL of the current build.

Reply:
Added a sed command to post_install_addon() to have package.mk store steamlink's version and hash.